src/server/items/validator.js

```diff
@@ -53,7 +52,6 @@ export default (req, res, next) => {
       message: 'wrong item address'
     }));
   }
-
   if (req.body.createdDate && !(typeof req.body.createdDate === 'string'
     && validator.isDate(req.body.createdDate))) {
     return next(new APIError(new Error(), {
```

Comment:
now we don't have to check createdDate and modifiedDate cause these are made in API func.